README.md

```diff
@@ -59,46 +59,46 @@ The [`config.yaml`](installer/config.yaml) file is structured to outline key com
 
 ```yaml
 ---
-rhtapCLI:
+cli:
   namespace: tssc
-  features: {}
+  products: {}
   dependencies: {}
 ```
 
-The attributes of the `rhtapCLI` object are as follows:
+The attributes of the `cli` object are as follows:
 
 - `.namespace`: Specifies the default namespace used by the installer, set to `tssc`. This namespace acts as the primary operational area for the installation process.
-- `.features`: Defines the features to be deployed by the installer. Each feature is identified by a unique name and a set of properties.
+- `.products`: Defines the features to be deployed by the installer. Each feature is identified by a unique name and a set of properties.
 - `.dependencies`: Specifies the dependencies rolled out by the installer in the specific order defined in the configuration file.
 
-## `rhtapCLI.features`
+## `cli.products`
 
-Defines the features the installer will deploy. Each feature is defined by a unique name and a set of properties. For instance, the following snippet defines a `featureName` block:
+Defines the products the installer will deploy. Each product is defined by a unique name and a set of properties. For instance, the following snippet defines a `productName` block:
 
 ```yaml
 ---
-rhtapCLI:
-  features:
-    featureName:
+cli:
```

Comment:
The root should be `tssc` or `tsscCLI`, as commented before.

Reply:
Fixed